foundations/html_css/css-foundations/intro-to-css.md

```diff
@@ -23,10 +22,10 @@ At the most basic level, CSS is made up of various rules. These rules are made u
 
 <div class="lesson-note" markdown="1">
 
+ 
```

Comment:
.

Reply:
Umm...what does that mean?

Reply:
I'm sorry, that comment was for me, hoping I could edit and commit that section myself and get this merge through. 
Could you please remove the extra line below the opening div for the note. There are currently two new lines, it should only be one.

Reply:
Sure

Reply:
Done